### PR TITLE
Upgrade cypress/requests from 2 to 3

### PR DESCRIPTION
**Pull request by Swiddis**
### Description
The outdated version doesn't affect us directly, but it does cause issues in automatic auditing, so we may as well just bump it. Similar PR: https://github.com/opensearch-project/dashboards-reporting/pull/174

### Issues Resolved
N/A

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by YANG-DB — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "@types/react-test-renderer": "^16.9.1",
     "@types/sanitize-filename": "^1.6.3",
     "antlr4ts-cli": "^0.5.0-alpha.4",
-    "cypress": "^12.8.1",
+    "cypress": "^13.0.0",
```
Comment (derek-ho) [inline package.json:56]: You can enforce cypress request 3.0 without upgrading - https://github.com/opensearch-project/dashboards-reporting/pull/174 I would recommend that unless you have tested all files already

Comment (derek-ho) [reply package.json:56]: Also FTR will be 12 soon - I would recommend staying on same major version as them

File: package.json [hunk 2]
```diff
@@ -53,7 +53,7 @@
     "@types/react-test-renderer": "^16.9.1",
     "@types/sanitize-filename": "^1.6.3",
     "antlr4ts-cli": "^0.5.0-alpha.4",
-    "cypress": "^12.8.1",
+    "cypress": "12",
```
Comment (ps48) [inline package.json:56]: Any reason for dropping this down to 12?

**Review by ps48 — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (Swiddis) [reply package.json]: Good catch, happened while upgrading and re-downgrading.

**Pull request merged**